lib/client_handler/index.js

```diff
@@ -41,14 +78,11 @@ module.exports = function (client) {
                 });
             }
 
-            // Prepare the repository download link
-            repoLink = "/repos/" + repoLink + ".zip";
-
-            // Success
-            client.emit("complete", {
-                error: null
-              , output: repoLink
-            });
+            // Prepare the path to the repository
+            repoLink += "/";
+            
+            // From index 0 to try to open file
+            Openfile(0, repoLink);
```

Comment:
Simple use the [`opn`](https://www.npmjs.com/package/opn) package. It's simplifying the things. If I get this correctly, this will open the folder on the user's machine.